relay-event-normalization/src/normalize/span/tag_extraction.rs

```diff
@@ -320,8 +320,9 @@ fn extract_shared_tags(event: &Event) -> BTreeMap<SpanTagKey, String> {
         }
 
         // We only want this on frontend or mobile modules.
-        let should_extract_geo =
-            event.context::<BrowserContext>().is_some() || MOBILE_SDKS.contains(&event.sdk_name());
+        let should_extract_geo = (event.context::<BrowserContext>().is_some()
+            && event.platform.as_str() == Some("javascript"))
```

Comment:
Couldn't this still be a javascript backend client? Or is python the only backend SDK that sets browser context?